Automattic/vip-cli
# [dev-env] Dynamic WordPress Image List

## Description
* On `vip dev-env create` The list of available WordPress Images will be produced dynamically based on a real time call to the images manifest on github.
* Will show a "Pre-Release" tag for images that are pre-release
* The default selection will always be the most recent stable release unless another version was selected.

![Screen Shot 2022-02-03 at 1 17 14 PM](https://user-images.githubusercontent.com/46167019/152422315-f3e978ba-cb67-4e2c-bfc8-8df83bb4102e.png)

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `vip dev-env create`
1. Verify  list of WordPress Images are up to date on the 3rd question of the form.
